portfolio.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" type="text/css" href="port_styles.css">
+</head>
+
+<header class="header">
+  <nav>
+    <a href="./index.html">Home</a>
+    <a href="./about.html">About Me</a>
+    <!-- <a href="mailto:dsorin@proton.me" class="button">Contact</a> -->
+  </nav>
+</header>
+
+<body id="porto">
+  <div class="container">
+    <div class="box" style="--clr:#89ec">
+      <div class="content">
+        <div class="icon"><ion-icon name="eye"></ion-icon></div>
+        <div class="text">
+          <h3>Viewing Party</h3>
+          <p>DESCRIPTION</p>
+          <a href="𝗵ttps://github.com/sorindevops/task-list-api">Viewing Party</a>
+        </div>
+      </div>
+    </div>
+    <div class="box" style="--clr:#eb5ae5">
+      <div class="content">
+        <div class="icon"><ion-icon name="list-circle"></ion-icon></div>
+        <div class="text">
+          <h3>Task List</h3>
+          <p>DESCRIPTION</p>
+          <a href="𝗵ttps://github.com/sorindevops/task-list-api">TasK List</a>
+        </div>
+      </div>
+    </div>
+    <div class="box" style="--clr:#5b98eb">
+      <div class="content">
+        <div class="icon" <ion-icon name="logo-wordpress"></ion-icon>></div>
+        <div class="text">
+          <h3>Swap Meet</h3>
+          <p>DESCRIPTION</p>
+          <a href="𝗵ttps://github.com/sorindevops/task-list-api">Swap Meet</a>
+        </div>
+      </div>
+    </div>
+    <div class="box" style="--clr:#89ec">
+      <div class="content">
+        <div class="icon" <ion-icon name="logo-wordpress"></ion-icon>></div>
+        <div class="text">
+          <h3>AdaGrams</h3>
+          <p>DESCRIPTION</p>
+          <a href="𝗵ttps://github.com/sorindevops/task-list-api">AdaGrams</a>
+        </div>
+      </div>
+    </div>
+<script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
```

Comment:
Not sure what instructions were for loading these but I'd suggest adding a link to the documentation, since it's much more common to load libraries in the document head unless directed otherwise.